packages/utilities/psammead-locales/moment/yo.test.js

```diff
@@ -0,0 +1,219 @@
+/* eslint-disable */
+import moment from 'moment';
+import './yo';
+
+moment.locale('yo')
+
+// This asset overrides the gunit assertion done in the moment codebase.
+// Format and styling of this file has been keep consistent with the official moment tests.
+// An example of these tests can be seen at https://github.com/moment/moment/blob/develop/src/test/locale/en-gb.js
+const assert = { equal: (val1, val2) => expect(val1).toEqual(val2) };
+
+test('parse', function () {
+    var tests = 'Sẹ́rẹ́ Sẹ́r_Èrèlè Èrl_Ẹrẹ̀nà Ẹrn_Ìgbé Ìgb_Èbibi Èbi_Òkùdu Òkù_Agẹmo Agẹ_Ògún Ògú_Owewe Owe_Ọ̀wàrà Ọ̀wà_Bélú Bél_Ọ̀pẹ̀̀ Ọ̀pẹ̀̀'.split('_'), i;
+    function equalTest(input, mmm, i) {
+        assert.equal(moment(input, mmm).month(), i, input + ' should be month ' + (i + 1));
+    }
+    for (i = 0; i < 12; i++) {
+        tests[i] = tests[i].split(' ');
+        equalTest(tests[i][0], 'MMM', i);
+        equalTest(tests[i][1], 'MMM', i);
+        equalTest(tests[i][0], 'MMMM', i);
+        equalTest(tests[i][1], 'MMMM', i);
+        equalTest(tests[i][0].toLocaleLowerCase(), 'MMMM', i);
+        equalTest(tests[i][1].toLocaleLowerCase(), 'MMMM', i);
+        equalTest(tests[i][0].toLocaleUpperCase(), 'MMMM', i);
+        equalTest(tests[i][1].toLocaleUpperCase(), 'MMMM', i);
+    }
+});
+
+test('format', function () {
+    var a = [
+            ['dddd, MMMM Do YYYY, h:mm:ss a', 'Àìkú, Èrèlè ọjọ́ 14 2010, 3:25:50 pm'],
+            ['ddd, hA', 'Àìk, 3PM'],
+            ['M Mo MM MMMM MMM', '2 ọjọ́ 2 02 Èrèlè Èrl'],
+            ['YYYY YY', '2010 10'],
+            ['D Do DD', '14 ọjọ́ 14 14'],
+            ['d do dddd ddd dd', '0 ọjọ́ 0 Àìkú Àìk Àì'],
+            ['DDD DDDo DDDD', '45 ọjọ́ 45 045'],
+            ['w wo ww', '6 ọjọ́ 6 06'],
+            ['h hh', '3 03'],
+            ['H HH', '15 15'],
+            ['m mm', '25 25'],
+            ['s ss', '50 50'],
+            ['a A', 'pm PM'],
+            ['[the] DDDo [day of the year]', 'the ọjọ́ 45 day of the year'],
+            ['LTS', '3:25:50 PM'],
+            ['L', '14/02/2010'],
+            ['LL', '14 Èrèlè 2010'],
+            ['LLL', '14 Èrèlè 2010 3:25 PM'],
+            ['LLLL', 'Àìkú, 14 Èrèlè 2010 3:25 PM'],
+            ['l', '14/2/2010'],
+            ['ll', '14 Èrl 2010'],
+            ['lll', '14 Èrl 2010 3:25 PM'],
+            ['llll', 'Àìk, 14 Èrl 2010 3:25 PM']
+        ],
+        b = moment(new Date(2010, 1, 14, 15, 25, 50, 125)),
+        i;
+    for (i = 0; i < a.length; i++) {
+        assert.equal(b.format(a[i][0]), a[i][1], a[i][0] + ' ---> ' + a[i][1]);
+    }
+});
+
+test('format ordinal', function () {
```

Comment:
Could we simplify this test. With a `for` loop maybe.

Reply:
Possibly, but this file has been copied from moment's tests, and I'd prefer to keep them as similar as possible. I can add a note explaining that to the top of the file instead?

Reply:
Yes. That would be better.